js/full-screen-picture.js

```diff
@@ -5,9 +5,24 @@ import {isEscapeKey} from './util.js';
 const bigPicture = document.querySelector('.big-picture');
 const bigPictureImg = document.querySelector('.big-picture__img');
 const socialComments = document.querySelector('.social__comments');
-const socialCommentsCount = document.querySelector('.social__comment-count');
 const commentsLoader = document.querySelector('.comments-loader');
 const body = document.querySelector('body');
+const commentVisible = document.querySelector('.social__comment-visible');
+const commentsCount = document.querySelector('.social__comment-count');
+
+//функция для выведения на экран пяти комментариев
+const showComments = () => {
+  const commentsMassive = socialComments.querySelectorAll('.hidden');
+  for (let i = 0; i < commentsMassive.length; i++) {
+    if(i < 5) {
+      commentsMassive[i].classList.remove('hidden');
+    }
+  }
+  if (commentsMassive.length < 5) {
+    commentsLoader.classList.add('hidden');
+  }
```

Comment:
```suggestion
  if (commentsMassive.length <= 5) {
    commentsLoader.classList.add('hidden');
  }
```
Иначе получаем небольшой баг при количестве комментов кратном 5. Показано `5 из 5` или `10 из 10` и т.д. комментов и при этом кнопка `Показать еще` видна. Попробуй сам)